sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 1.0.0b13 (Unreleased)
+
+### Breaking changes
+
+* Updated evaluator ids for AOAI graders in `_patch_evaluations.py` to reach format consistency with Foundry evaluator ids; Added SCORE_MODEL_GRADER;
```

Comment:
No need to mention source file names... focus on the impact on the developer using the package.